commands/scan.md

```diff
@@ -142,7 +142,7 @@ As you can see most of the calls returned zero elements, but the last call where
 
 ## The TYPE option
 
-As of version 6.0 you can use this option to ask `SCAN` to only return objects that match a given `type`, allowing you to iterate through the database looking for keys of a specific type. The **TYPE** option is only available on the whole-database `SCAN`, not `HSCAN` or `ZSCAN` etc.
+You can use the `!TYPE` option to ask `SCAN` to only return objects that match a given `type`, allowing you to iterate through the database looking for keys of a specific type. The **TYPE** option is only available on the whole-database `SCAN`, not `HSCAN` or `ZSCAN` etc.
```

Comment:
what's with the `!`?

Reply:
This is in order to "escape" the thing so it doesn't auto-link to the Redis `TYPE` command here.